file.py

```diff
@@ -1 +1,2 @@
-Text to be changed
+for i in range(100):
+    print("good")
```

Comment:
to jest totalnie do dupy

Reply:
no nie wiem co o tym myśleć, starałem się ;)